tests/fixtures/issues/issue-1219.php

```diff
@@ -6308,59 +6308,6 @@
         'lite' => false,
         'standard' => false,
     ],
-    'issue-1219-119' => [
-        'ua' => 'Mozilla/5.0 (Windows; U; Windows NT 6.1; en-GB; rv:1.9.1.17) Gecko/20110123 Mozilla/5.0 (Windows; U; Windows NT 5.1; en-US; rv:1.8.1.2) Gecko/20070225 lolifox/0.32',
```

Comment:
This Ua is one of the samples you posted to issue #1219. Did you forgot a linebreak there?